.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
out of curiosity, what does this do?

Reply:
This basically stops there being a problem between LF and CRLF variations of the code, which arises when the same code is pushed and pulled between different OSs